pinakes/settings/defaults.py

```diff
@@ -413,3 +423,21 @@
 CORS_ALLOW_CREDENTIALS = False
 CSRF_TRUSTED_ORIGINS = env.list("PINAKES_CSRF_TRUSTED_ORIGINS", default=[])
 SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+
+# Metrics collections
+PINAKES_INSIGHTS_TRACKING_STATE = env.bool(
+    "PINAKES_INSIGHTS_TRACKING_STATE", False
+)
+PINAKES_INSIGHTS_URL = env.str("PINAKES_INSIGHTS_URL", default="ingress_url")
```

Comment:
@hsong-rh Can we find out what should the default value be? I think someone was trying to create a upload url and Kafka topic for us.